Gordon360/Controllers/NewsController.cs

```diff
@@ -187,17 +187,17 @@ public ActionResult<StudentNews> Delete(int newsID)
         /// (Controller) Edits a news item in the database
         /// </summary>
         /// <param name="newsID">The id of the news item to edit</param>
-        /// <param name="newData">The news object that contains updated values</param>
+        /// <param name="studentNewsEditting">The news object that contains updated values</param>
```

Comment:
```suggestion
        /// <param name="studentNewsUpload">The news object that contains updated values</param>
```
Editing is misspelled in the three locations here.

Also, I personally think it's cleaner to name your object variables after a noun rather than a verb. Perhaps `studentNewsUpload` or, if you want to stick with edit, `studentNewsEdit` instead of the participle, `studentNewsEditing`.